dapp/src/app/actions/colony.ts

```diff
@@ -0,0 +1,54 @@
+const { providers, Wallet } = require("ethers");
+const { default: EthersAdapter } = require("@colony/colony-js-adapter-ethers");
+const { TrufflepigLoader } = require("@colony/colony-js-contract-loader-http");
+
+const { default: ColonyNetworkClient } = require("@colony/colony-js-client");
+const loader = new TrufflepigLoader();
+const provider = new providers.JsonRpcProvider("http://localhost:8545/");
```

Comment:
Same comment about metamask.

Reply:
For providers we should support:
- Local host provider
- Metamask
- Local node provider (geth running)
- Other?

I'll create a task to look into this further.

Reply:
https://github.com/JingyuZ/GithubColony/issues/18